site/app/Http/Controllers/TagController.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Course;
+use App\Http\Requests\IndexTag;
+use App\Http\Requests\StoreTag;
+use App\Http\Requests\UpdateTag;
+use App\Tag;
+use Illuminate\Support\Facades\Auth;
+
+class TagController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return Renderable
+     *
+     * @throws AuthorizationException
+     */
+    public function index(Course $course, IndexTag $request)
+    {
+        $this->authorize('viewInCourseConfiguration', [Tag::class, $course]);
```

Comment:
Renamed but not wrote a comment since it's the same as the create, update and delete policies (we did not write "Create a tag in the course configuration"...).